app/assets/stylesheets/pages.css.scss

```diff
@@ -319,25 +319,47 @@ $include-html-global-classes: false;
   }
 }
 
-.rackspace {
+.footer {
   background-color: $hackedu-black;
-  color: $text-on-black;
-  padding: 10px 0 10px 0;
-
-  p {
-    font-size: rem-calc(14);
-    margin: 7.5px 0 7.5px 0;
+  color: $hackedu-white;
+  font-weight: 300;
+  padding: 30px 0;
 
-    @media #{$medium-up} {
+  .nav {
+    h1 {
+      color: $hackedu-white;
       font-size: rem-calc(18);
-      margin: 15px 0 15px;
+      font-weight: 600;
+      margin-left: 5px;
+      text-transform: uppercase;
     }
 
-    a {
-      color: scale-color($text-on-black, $lightness: 10%);
-      text-decoration: underline;
+    ul {
+      margin-left: 0;
+
+      li {
+        color: $hackedu-white;
+        font-weight: inherit;
+        list-style-type: none;
+        padding: 5px;
+
+        a {
+          color: $hackedu-white;
+          display: block;
+          height: 100%;
+          width: 100%;
+        }
+      }
+
+      .link:hover {
```

Comment:
Same thing as above applies here. I think we'd need a much larger redesign to follow this guideline